app/src/main/res/values/colors_material3.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright © 2014-2021 The Android Password Store Authors. All Rights Reserved.
+  ~ SPDX-License-Identifier: GPL-3.0-only
+  -->
+
+<resources>
+  <color name="md_theme_light_primary">#006591</color>
+  <color name="md_theme_light_onPrimary">#FFFFFF</color>
+  <color name="md_theme_light_primaryContainer">#C7E6FF</color>
+  <color name="md_theme_light_onPrimaryContainer">#001E30</color>
+  <color name="md_theme_light_secondary">#4F606E</color>
+  <color name="md_theme_light_onSecondary">#FFFFFF</color>
+  <color name="md_theme_light_secondaryContainer">#D3E5F5</color>
+  <color name="md_theme_light_onSecondaryContainer">#0B1D29</color>
+  <color name="md_theme_light_tertiary">#006494</color>
+  <color name="md_theme_light_onTertiary">#FFFFFF</color>
+  <color name="md_theme_light_tertiaryContainer">#C8E6FF</color>
+  <color name="md_theme_light_onTertiaryContainer">#001E31</color>
+  <color name="md_theme_light_error">#BA1B1B</color>
+  <color name="md_theme_light_errorContainer">#FFDAD4</color>
+  <color name="md_theme_light_onError">#FFFFFF</color>
+  <color name="md_theme_light_onErrorContainer">#410001</color>
+  <color name="md_theme_light_background">#FCFCFF</color>
+  <color name="md_theme_light_onBackground">#1A1C1E</color>
+  <color name="md_theme_light_surface">#FCFCFF</color>
+  <color name="md_theme_light_onSurface">#1A1C1E</color>
+  <color name="md_theme_light_surfaceVariant">#DDE3EA</color>
+  <color name="md_theme_light_onSurfaceVariant">#41474D</color>
+  <color name="md_theme_light_outline">#72787E</color>
+  <color name="md_theme_light_inverseOnSurface">#F0F0F3</color>
```

Comment:
nit: can we rename these to onSurfaceInverse and surfaceInverse to follow the pattern?